README.md

```diff
@@ -1,87 +1,14 @@
 # MySQL with automatic failover on Kubernetes
 
-Our requirement was an SQL server for modest loads
-but with high availability because it backs our login system
-and public web.
+This is a galera cluster setup, with plain manifests.
+We actually use it in production, though with modest loads.
 
-We want automatic failover, not manual, and a single configuration to maintain, not leaders and followers.
+## Get started
 
-Initially we tried Postgres, but clustering solutions like [pgpool-II]() felt a bit outdated, and [patroni](https://github.com/zalando/patroni) etc was overly complex for our needs.
+First create a storage class `mysql-data`. See exampels in `./configure/`.
```

Comment:
Did GKE ever implement some kind support for automagic creation of disks so that no gcloud command is necessary to bootstrap this from zero?

Reply:
Yes, but in practice it seems you always want storage classes. I think minikube's automatic volume provisioning won't happen without it. And anyway, it makes sense to decide if we want SSD or not, and [Retain](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/) or not.

Bootstrap didn't get easier, but scaling up to additional instances is only a kubectl scale now, whereas previously we had to create volumes.